docs/admin/pool_operations.md

```diff
@@ -191,174 +191,382 @@ said to be operating in "phase-2" mode.
+
+Creating DAOS pool with 100% of all storage
+Pool created with 8.65%,91.35% storage tier ratio
 -------------------------------------------------
-  UUID             : bad54f1d-8976-428b-a5dd-243372dfa65c
+  UUID             : ee7af142-3d72-45bf-9dc2-e1060c0de5be
   Service Leader   : 1
   Service Ranks    : [0-1]
   Storage Ranks    : [0-1]
-  Total Size       : 2.4 TB
-  Metadata Storage : 140 GB (70 GB / rank)
-  Data Storage     : 2.2 TB (1.1 TB / rank)
-  Memory File Size : 140 GB (70 GB / rank)
+  Total Size       : 3.0 TB
+  Metadata Storage : 258 GB (129 GB / rank)
+  Data Storage     : 2.7 TB (1.4 TB / rank)
+  Memory File Size : 129 GB (64 GB / rank)
+```
+
+
+7. A larger pool with 6 engines/ranks across 3 hosts using the same shared-role
+configuration and pool-create commandline as the previous example.
+
+```bash
+$ dmg -i pool create bob -z 100% --mem-ratio 50%
 
+Creating DAOS pool with 100% of all storage
+Pool created with 8.65%,91.35% storage tier ratio
+-------------------------------------------------
+  UUID             : 678833f3-ba0a-4947-a2e8-cef45c3c3977
+  Service Leader   : 3
+  Service Ranks    : [0-1,3-5]
+  Storage Ranks    : [0-5]
+  Total Size       : 8.9 TB
+  Metadata Storage : 773 GB (129 GB / rank)
+  Data Storage     : 8.2 TB (1.4 TB / rank)
+  Memory File Size : 386 GB (64 GB / rank)
 ```
 
-Rough calculations: 1.2TB of usable space is returned from storage scan and
-because roles are shared required META (70GB) is reserved so only 1.1TB is
-provided for data.
+Here the size has increased linearly with the per-rank sizes remaining the
+same.
+
+
+8. Now for a more involved example with shared roles. Create two pools of
+roughly equal size each using half available capacity and a `--mem-ratio` of
+50%.
+
+An administrator can use the `dmg storage query usage` command to gauge
+available capacity across ranks and tiers. Adding `--show-usable` flag shows
+capacity that could be used to store DATA once META overheads of a new pool
+have been taken into account:
 
-Logging shows:
 ```bash
-DEBUG 2024/09/24 15:44:38.554431 pool.go:1139: added smd device c7da7391-9077-4eb6-9f4a-a3d656166236 (rank 1, ctrlr 0000:d8:00.0, roles "data,meta,wal") as usable: device state="NORMAL", smd-size 623 GB (623307128832), ctrlr-total-free 623 GB (623307128832)
-DEBUG 2024/09/24 15:44:38.554516 pool.go:1139: added smd device 18c7bf45-7586-49ba-93c0-cbc08caed901 (rank 1, ctrlr 0000:d9:00.0, roles "data,meta,wal") as usable: device state="NORMAL", smd-size 554 GB (554050781184), ctrlr-total-free 1.2 TB (1177357910016)
-DEBUG 2024/09/24 15:44:38.554603 pool.go:1246: based on minimum available ramdisk capacity of 70 GB and mem-ratio 1.00 with 70 GB of reserved metadata capacity, the maximum per-rank sizes for a pool are META=70 GB (69792169984 B) DATA=1.1 TB (1107565740032 B)
+$ dmg -i storage query usage --show-usable --mem-ratio 50% -l wolf-[310-312]
+
+Tier Roles
+---- -----
+T1   data,meta,wal
+
+Rank T1-Total T1-Usable T1-Usage
```

Comment:
Explanation of the last columns meaning could be helpful.  I am not sure if it is the calculated according to the available size or the usable one